app/models/flood_risk_engine/reference_number.rb

```diff
@@ -0,0 +1,28 @@
+module FloodRiskEngine
+  class ReferenceNumber < ActiveRecord::Base
+    PREFIX = "EXFRA".freeze
+    PADDING = "0".freeze
+    OFFSET = 10_000
```

Comment:
According to original story and a meeting today about go live tasks, the expectation is this is 1000 not 10000